frontend-beeyond/src/app/app-routing.module.ts

```diff
@@ -14,25 +16,45 @@ const routes: Routes = [
   {
     path: 'accounting',
     loadChildren: () =>
-      import('./modules/accounting/accounting.module').then(m => m.AccountingModule)
+      import('./modules/accounting/accounting.module').then(m => m.AccountingModule),
+    canActivate: [AuthGuard],
+    data: {
```

Comment:
All these `data`s are not required. You can also group [guards](https://stackoverflow.com/a/43487962/11125147)